### PR TITLE
[WIP] New Website: New Docs [1/3]

### PR DESCRIPTION
**Pull request by ThomasDelteil**
Adding new files for the new website

Website can be seen here: https://thomasdelteil.github.io/mxnet.io-v2/

- `docs/python_docs`: New python website, a port + cosmetic changes from https://github.com/mli/new-docs
- `docs/static_site`: New Jekyll static site, with the tutorials from other languages and content from old website ported to the new format
- `docs/cpp_docs`: Folder to hold on the Doxyfile and Makefile to build the CPP docs.

Known issues:
- get_started/ doesn't work with current .htaccess
- python tutorials htaccess rule need to be revisited
- blinking on python API, should hide entire content

### CODE REVIEW

File: docs/static_site/src/pages/get_started/windows_setup.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+layout: page
+title: Windows Setup
+action: Get Started
+action_url: /get_started
+permalink: /get_started/windows_setup
+---
+
+# Installing MXNet on Windows
+
+The following describes how to install with pip for computers with CPUs, Intel CPUs, and NVIDIA GPUs. Further along in the document you can learn how to build MXNet from source on Windows, or how to install packages that support different language APIs to MXNet.
```
Comment (vishaalkapoor) [inline docs/static_site/src/pages/get_started/windows_setup.md]: nit: you'll learn how to build MXNet ..., and how to install (and vs or)

**Comment by pengzhao-intel** [comment]
@ThomasDelteil do we have a time schedule for website changing?
We're still working on improvements based on the current website and I think we need to align with the new website.
What's your opinion?

**Comment by ThomasDelteil** [comment]
Hey @pengzhao-intel, thanks for reaching out! :)
We are trying to get this merged as soon as possible, aiming for 31st of August launch date.
It should be quite easy to add new content to the new website. 
Have a look at the `static_website/src/pages` folder and I can also help you port whatever content you wanted to build. We can also look into having a new page or new sections.

**Comment by pengzhao-intel** [comment]
Thanks, @ThomasDelteil 

@xinyu-intel @TaoLv please take a look at the new website and we need to move our documents in new home :)

**Comment by pengzhao-intel** [comment]
@sandeep-krishnamurthy do we have the plan to put the unified dashboard #15757 into the new website?
If so, the user will find the latest performance on the website. Maybe includes GluonCV/NLP/TS, DGL in the future as well. 
Current [perf page](https://mxnet.incubator.apache.org/versions/master/faq/perf.html) is really out-of-date and we are refreshing it now.

**Comment by aaronmarkham** [comment]
I fixed one license issue, but now another popped up. I think that RAT checker just bails once it finds one issue instead of telling you the totality of files that need licenses. I'm going to ignore this check for now.

**Comment by ElaineBao** [comment]
Hi, is it possible to make API docs easier to find ? I have to click many times to get to the function page: Main page -> Docs & Tutorials -> Python API Reference -> Python API -> mxnet.ndarray -> NDArray -> mxnet.ndarray.NDArray.dtype.

File: docs/python_docs/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# README
```
Comment (aaronmarkham) [inline docs/python_docs/README.md]: Should update this...

**Comment by aaronmarkham** [comment]
I have updates pending for the license headers.

**Comment by aaronmarkham** [comment]
@ThomasDelteil I think that the static site should get its own license file originating from whatever theme or source this started from... otherwise each file in that subdir is going to need licenses. Seems easier to just drop a LICENSE file in there... wdyt? For now, I'm excluding scanning docs/static_site, going with the assumption that this will be true. I did this for the theme because it has a license file in there.

**Comment by aaronmarkham** [comment]
CI fails the sanity test due to RAT license check.
I've excluded the offending folder here:
https://github.com/apache/incubator-mxnet/pull/15884/commits/aa74a30ae2597b61ed9d0571baf4a0d8eec3b655#diff-ec188843973709ab04de4ff565735aecR10
But I get an error here:
http://jenkins.mxnet-ci.amazon-ml.com/job/mxnet-validation/job/sanity/job/PR-15884/9/display/redirect

**Comment by aaronmarkham** [comment]
> CI fails the sanity test due to RAT license check.
> I've excluded the offending folder here:
> [aa74a30#diff-ec188843973709ab04de4ff565735aecR10](https://github.com/apache/incubator-mxnet/commit/aa74a30ae2597b61ed9d0571baf4a0d8eec3b655#diff-ec188843973709ab04de4ff565735aecR10)
> But I get an error here:
> http://jenkins.mxnet-ci.amazon-ml.com/job/mxnet-validation/job/sanity/job/PR-15884/9/display/redirect

Leaving this here for reference... come to find out, you cannot exclude subfolders. The rat checker only seems to accept one folder, like `_build/*` but not `_build/xml/*`. Really quite irritating and over broad. I tried variety of regexes to no avail. Whatever the case, the existing rat-exclude file is really pretty broad and probably excludes way too many things. I added some comments to it and tried to get it as specific as possible for these new items.

**Comment by vishaalkapoor** [comment]
Approved!

**Pull request merged**